cluster/images/conformance/conformance-e2e.yaml

```diff
@@ -0,0 +1,71 @@
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: conformance
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  labels:
+    component: conformance
+  name: conformance-serviceaccount
+  namespace: conformance
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  labels:
+    component: conformance
+  name: conformance-serviceaccount-role
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: conformance-serviceaccount
+subjects:
+- kind: ServiceAccount
+  name: conformance-serviceaccount
+  namespace: conformance
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels:
+    component: conformance
+  name: conformance-serviceaccount
+rules:
+- apiGroups:
+  - '*'
+  resources:
+  - '*'
+  verbs:
+  - '*'
+- nonResourceURLs:
+  - '/metrics'
+  - '/logs'
+  - '/logs/*'
+  verbs:
+  - 'get'
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: e2e-conformance-test
+  namespace: conformance
+spec:
+  containers:
+  - name: conformance-container
+    image: staging-k8s.gcr.io/conformance-amd64:v1.14-alpha4
```

Comment:
do we want to hardcode this version in here?

also why staging-k8s?

Reply:
will fix. these are from the code that already exists, will have to change that